javaScript-week1/Housey-pricey.js

```diff
@@ -0,0 +1,30 @@
+// Housey pricey
+
+function sumVolume(width, height, depth) {
+    return width * height * depth;
+  }
+  
+  function trueHousePrice(volumeInMeters, gardenSize) {
+    return volumeInMeters * 2.5 * 1000 + gardenSize * 300;
+  }
+  
+  function compare(truePrice, costPrice) {
+    if (truePrice > costPrice) {
+      console.log(`paying too little. The real price is: ${truePrice}`);
+    } else {
+      console.log(
+        `paying more than the estimated cost. The real price is: ${truePrice}`
+      );
+    }
+  }
+  const marketPricePeter = 2.5e6;
+  const marketPriceJulia = 1e6;
```

Comment:
I see this way of writing numbers for the first time. I see it works, but it's not the clearest and easiest. Could you share some article or blog post explaining it?

Reply:
https://www.youtube.com/watch?v=0ELC8R69Rkw&list=PL7TLF4T4Tq2TtNmadzRfxYIB9683uhpbD&index=10 
this is a link to the video.